app/models/intake.rb

```diff
@@ -580,4 +580,13 @@ def update_or_create_additional_consent_pdf
   def might_encounter_delayed_service?
     vita_partner.at_capacity?
   end
+
+  # Backfills current_step for clients who started intake before we tracked current_step
+  def determine_current_step
+    return current_step if current_step.present?
+
+    step = QuestionNavigation.determine_current_step(self)
+    update(current_step: step)
+    step
+  end
```

Comment:
maybe we can move this into a rake task

Reply:
My take was that there are so many unused/unusable intakes, so we'd be looping through a bunch of extraneous data in order to accomplish the backfill. Here, we only need to run the backfill for clients who actually decide to use the returning client feature. This is intended to be removed at the end of the tax season, because moving forward we'll already keep track of it throughout the flow. My intention here, like indicated in the ticket, was to implement something fairly efficiently without having to spend additional time creating a query and running it on prod. I'm willing to be convinced otherwise, but it just seems like running this on every intake would mean having to split things into groups, consider which intakes to run it on etc etc.